Automattic/wp-calypso
# People: Invites: Display notices after redirect

__How to test__

- Checkout `update/invite-notices` branch
- Go to `$site/wp-admin/users.php?page=wpcom-invite-users`
- Invite a user
- Log in as the user
- In the invite email, make note of the invitation key in the activation link
- Go to `/accept-invite/$site/$invite_key`
- Accept the invite
- Assert that the notice appears only after being redirected to the site posts list
